x-pack/plugins/cases/docs/openapi/components/schemas/user_properties.yaml

```diff
@@ -6,4 +6,7 @@ full_name:
   example: null
 username:
   type: string
-  example: elastic
\ No newline at end of file
+  example: elastic
+profile_uid:
```

Comment:
@lcawl this field is optional, so it might not be returned in the responses. What's the best way to capture that? Or is that not necessary? Should I use `nullable`? Technically it won't be set to `null` it just won't be returned in the response.